lib/events.js

```diff
@@ -246,14 +246,11 @@ EventEmitter.prototype.addListener = function addListener(type, listener) {
       m = $getMaxListeners(this);
       if (m && m > 0 && existing.length > m) {
         existing.warned = true;
-        if (!internalUtil)
-          internalUtil = require('internal/util');
-
-        internalUtil.error('warning: possible EventEmitter memory ' +
-                           'leak detected. %d %s listeners added. ' +
-                           'Use emitter.setMaxListeners() to increase limit.',
-                           existing.length, type);
-        console.trace();
+        if (!badPracticeWarning)
+          badPracticeWarning = require('internal/warning').badPracticeWarning;
+        badPracticeWarning('Possible EventEmitter memory leak detected. ' +
+                           `${existing.length} ${type} listeners added. ` +
+                           'Use emitter.setMaxListeners() to increase limit');
```

Comment:
It could very well be. We can simply drop the badPractice Warning distinction entirely.